### PR TITLE
fix: changed empty team member message

### PR DESCRIPTION
**Pull request by Shivam327**
Earlier:
image

After updating (New Message):
image

### CODE REVIEW

File: desk/src/pages/desk/team/TeamSingle.vue [hunk 1]
```diff
@@ -43,7 +43,9 @@
                 </template>
               </Button>
             </div>
-            <div v-else class="text-base text-gray-900">🙇 Such empty</div>
+            <div v-else class="text-base text-gray-900">
+              No Members found in Team: {{ teamId }}
```
Comment (ssiyad) [inline desk/src/pages/desk/team/TeamSingle.vue]: User lowercase - `No members found in team`

Comment (Shivam327) [reply desk/src/pages/desk/team/TeamSingle.vue]: @ssiyad Done, changed to lower case

Comment (Shivam327) [reply desk/src/pages/desk/team/TeamSingle.vue]: please check its done

File: desk/src/pages/desk/team/TeamSingle.vue [hunk 2]
```diff
@@ -43,7 +43,9 @@
                 </template>
               </Button>
             </div>
-            <div v-else class="text-base text-gray-900">🙇 Such empty</div>
+            <div v-else class="text-base text-gray-900">
+              no members found in team: {{ teamId }}
```
Comment (ssiyad) [inline desk/src/pages/desk/team/TeamSingle.vue]: Capitalise first letter (of first word) `no` -> `No`

Comment (Shivam327) [reply desk/src/pages/desk/team/TeamSingle.vue]: Done

**Review by ssiyad — APPROVED** [review]

**Pull request merged**